generator/config/accumulator/accumulator.yaml

```diff
@@ -1,6 +1,6 @@
 # $schema: ../schema.json
 name: $accumulator
-link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/accumulator/'
```

Comment:
Is it possible to keep quotes around strings? This style changes make the diff hard to read. Also, I use quotes by default around every string to prevent unexpected parsing rule from Yaml.

Reply:
Yeah, at least `prettier` supports YAML out of the box, so that feels like a great way to remove any formatting preference ambiguity